modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/connections/HostConnections.java

```diff
@@ -104,10 +112,9 @@ public NHttpClientConnection getConnection() {
                 long currentTime = System.currentTimeMillis();
                 long connectionInitTime = (Long) conn.getContext().getAttribute(PassThroughConstants.
                                                                                         CONNECTION_INIT_TIME);
-                long lastReleasedTime = (Long) conn.getContext().getAttribute(PassThroughConstants.
-                                                                                      CONNECTION_RELEASE_TIME);
-                if (isMaximumLifeSpanExceeded(currentTime, connectionInitTime) || isIdleTimeExceeded(currentTime,
```

Comment:
Shall we delete this isIdleTimeExceeded method as well and clean the code.

Reply:
Fixed